worker/stemworker/socketio/__init__.py

```diff
@@ -73,16 +86,28 @@ def get_worker_files(path):
         logger.info('stem.pipeline.execute: %s' % params)
         pipeline_id = params['pipelineId']
         pipeline = get_pipeline_instance(pipeline_id)
-        files = get_worker_files(params['params']['path'])
-        if (len(files) == 0):
+
+        path_hdf5 = params['params'].get('path_hdf5')
+        path = params['params'].get('path')
```

Comment:
Not really, in this use case the path is a path to a file from a local girder assetstore, and it would look somethig like: `/home/alessandro/girder_fs/80/9f/809f13e0ba002ca98291ed5e6cef6ffe15098ba09fd453cd17212cc693a059758bda4bee92fa39d5b25a07b0e6f986822b15311b992c49cb899777e2bd1d9b0e`.
I'm open to suggestions if you can think of a way to unify it though

Reply:
Ok, in that case it might be better to just added another property that indicates the format, then at least we only have one `path`.

Reply:
the formats would be `h5` or `dat`

Reply:
Done